spec/features/add_an_ip_address_spec.rb

```diff
@@ -24,16 +24,31 @@
 
     context 'and that IP is valid' do
       before do
+        ENV['S3_PUBLISHED_LOCATIONS_IPS_BUCKET'] = 'StubBucket'
+        ENV['S3_PUBLISHED_LOCATIONS_IPS_OBJECT_KEY'] = "StubKey"
```

Comment:
Not sure if double quotes here will fail gov-lint?

Reply:
They don't but good catch, will amend.